lib/json.ps1

```diff
@@ -146,19 +148,58 @@ function normalize_values([psobject] $json) {
     # Iterate Through Manifest Properties
     $json.PSObject.Properties | ForEach-Object {
 
+        $value = $_.Value
+        # Recursively edit psobjects
+        # If the values is psobjects, its not normalized
+        # For Example if manfiest have architecture and it's architecture have array with single value it's not formated
```

Comment:
Typos corrected: `For example if manifest have architecture and it's architecture have array with single value it's not formatted`